dev/org.eclipse.codewind.core/src/org/eclipse/codewind/core/internal/InstallUtil.java

```diff
@@ -213,11 +213,12 @@ public static void createProject(String name, String path, String url, IProgress
 		}
 	}
 	
-	public static ProjectInfo validateProject(String name, String path, IProgressMonitor monitor) throws IOException, JSONException, TimeoutException {
+	public static ProjectInfo validateProject(String name, String path, String hint, IProgressMonitor monitor) throws IOException, JSONException, TimeoutException {
```

Comment:
I would really like to get rid of ProjectInfo, ProjectType and ProjectLanguage.  All that is really needed is a mapping from the type id or language id to a display name (would be nice if PFE returned names that can be displayed).  But this is probably too big a change for now.  I can do it after 0.5.0.